src/shared/components/events-list/index.js

```diff
@@ -28,9 +28,19 @@ const EventsList = ({
         <ul className={styles.eventsList}>
           {
             relevantEvents.map((event) => (
-              <EventsListEntry
-                event={event}
+              <EventsNewsListEntry
+                id={event.id}
+                tags={event.tags}
+                slug={event.slug}
+                title={event.title}
+                strapline={event.strapline}
+                startDateTime={event.startDateTime}
+                endDateTime={event.endDateTime}
+                externalLinks={event.externalLinks}
+                internalLinks={event.internalLinks}
+                featureImageFilename={event.featureImageFilename}
```

Comment:
Can we just spread on the whole event? Or do we prefer explicitly listing them like this? I can see value in both.